app/scripts/filters/resources.js

```diff
@@ -320,7 +320,7 @@ angular.module('openshiftConsole')
         return routeWebURLFilter(route, host);
       }
       var label = (host || routeHostFilter(route));
```

Comment:
honestly i don't have a strong preference either way, i'm leaning towards dropping it just so its very obvious your route is not currently in a usable state